### PR TITLE
Jamie/bug table sweep

### PR DESCRIPTION
**Pull request by SEAjamieD**
### :nut_and_bolt: Description: What code changed, and why?
Additional sweep on formatting that may need to be adjusted related to https://github.com/chef/automate/pull/2963

### :chains: Related Resources
https://github.com/chef/automate/pull/2963

### :+1: Definition of Done
1. table headers match alignment with table columns
2. Any outstanding long content is ellipsed or contained in column

### :athletic_shoe: How to Build and Test the Change
build components/automate-ui && start_all_services
make serve

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Comment by SEAjamieD** [comment]
@msorens / @susanev - I've done an additional sweep through automate and made any adjustments I could find.  Most of this is shrinking down the table headers that match to the control menu since it was shrunk down.

I'm also going to move up the new table component swapping in priority - I started to scope creep on this and decided to back off and focus on the table swapping on its own branch.

File: components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list-row.html [hunk 1]
```diff
@@ -6,17 +6,17 @@
...
+  <chef-table-cell>{{ credentialsLogic.formatKeyType(credential.type) }}</chef-table-cell>
+  <chef-table-cell>{{ getLastModified(credential.last_modified) }}</chef-table-cell>
+  <chef-table-cell class="three-dot-column">
     <mat-select panelClass="chef-control-menu">
       <mat-option (onSelectionChange)="startCredentialDelete()" data-cy="delete">Remove Credential</mat-option>
     </mat-select>
-  </chef-td>
-</chef-tr>
+  </chef-table-cell>
+</chef-table-row>
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list-row.html]: this was scope creep - but i'm going to move forward with this type of swapping on its own branch.

File: components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list.html [hunk 2]
```diff
@@ -35,7 +35,7 @@
                   Last Modified
                   <chef-sort-toggle sort="last_modified" [order]="orderFor('last_modified')"></chef-sort-toggle>
                 </chef-th>
-                <chef-th></chef-th>
+                <chef-th class="three-dot-column"></chef-th>
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list.html]: applying this class matches the new width changes of the control menu cells that were just made.

File: components/automate-ui/src/app/pages/notifications/notifications.component.scss [hunk 3]
```diff
@@ -31,6 +31,16 @@
...
+  chef-td {
+    display: flex;
+    flex-direction: row;
+
+    a {
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/pages/notifications/notifications.component.scss]: This might end up being a stop gap until I swap out the tables, however this table content is unique in that it has two items inside its table-cell, which requires a little bit different of CSS.

**Comment by SEAjamieD** [comment]
This branch is an extension of https://github.com/chef/automate/issues/2962

**Comment by susanev** [comment]
since you are here on scan jobs, it would be really rad to remove that sorta informational table row, and move the button to the top of the table. the button label should adjust to `Create Scan Job` as well.
new-scan-jobs

same with nodes added tab
new-nodes-added

**Comment by SEAjamieD** [comment]
Any extras here can/will be handled in the table sweep i'm doing now: https://github.com/chef/automate/issues/2064
